custom-words.txt

```diff
@@ -2158,4 +2158,16 @@ zipdeploy
 ziplist
 Zoho
 zoneinfo
-zset
\ No newline at end of file
+zset
+classificationrules
```

Comment:
Do we want to keep this file sorted alphabetically?

Reply:
Yes please